.github/workflows/deploy-prd-enhanced-cache.yml

```diff
@@ -20,7 +29,9 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
       - name: Rebuild Cache if New Snooty Parser Version
-        uses: mongodb/docs-worker-actions/rebuild-parse-cache@main
+        uses: mongodb/docs-worker-actions/rebuild-parse-cache@cache-update-hotfix
```

Comment:
NOTE: revert this after merging [this PR](https://github.com/mongodb/docs-worker-actions/pull/80)